src/Entity/User.php

```diff
@@ -301,4 +302,31 @@ public function __toString()
     {
         return $this->getAlias() ?: $this->getUsername();
     }
+
+    /**
+     * @ORM\Column(name="googleAuthenticatorSecret", type="string", nullable=true)
+     */
+    private $googleAuthenticatorSecret;
```

Comment:
is that only for the google authenticator or can other products be used as well?

Reply:
I think it's copied from here https://github.com/scheb/two-factor-bundle/blob/master/Resources/doc/providers/google.md#basic-configuration and the `Scheb\TwoFactorBundle\Model\Google\TwoFactorInterface` is written for google authenticator. 

But as far as I can see, it is here to store TOTP/HOTP secrets. The use of other TOTP apps (e.g. https://github.com/andOTP/andOTP) should be possible too.

Reply:
@lduer is right. I added this property and column per the bundle's instructions, but the bundle also support email 2fa as well.

Reply:
then please rename the column accordingly, something like `twoFactorSecret`?

Reply:
@kevinpapst - I'm afraid not. The author of the bundle states in the docs that specific interfaces should be used per each authenticator module. However, per https://github.com/scheb/two-factor-bundle/blob/master/Resources/doc/providers/custom.md we could implement a generic one...